admin/class-export.php

```diff
@@ -207,17 +211,66 @@ private function write_file() {
 	/**
 	 * Zips the settings ini file
 	 *
-	 * @return boolean unsigned
+	 * @return bool|null
 	 */
 	private function zip_file() {
```

Comment:
Since this is a private function, we can rename it to something more to what it's doing now like `serve_settings_export`